src/sympc/tensor/grads/grad_functions.py

```diff
@@ -632,6 +633,40 @@ def backward(ctx: Dict[str, Any], grad: MPCTensor) -> MPCTensor:
         return grad.reshape(shape)
 
 
+class GradReLU(GradFunc):
+    """The sigmoid gradient function."""
+
+    @staticmethod
+    def forward(ctx: Dict[str, Any], x: MPCTensor) -> MPCTensor:
+        """Perform the feedforward and compute the result for the ReLU operation.
+
+        Args:
+            ctx (Dict[str, Any]): Context used to save information needed in the backward pass
+            x (MPCTensor): The operand on which to apply the sigmoid function
+
+        Returns:
+            relu(x) (MPCTensor): The sigmoid approximation applied on the input
+        """
+        mask = APPROXIMATIONS["sign"](x.gt(0.0))
```

Comment:
MAYBE: Call ```relu``` from `torch.module.nn.functional`? (or do you have any reason to do it like this?)

Reply:
I need to store the mask in ctx for a backward pass if I use relu straight I won't have the mask.

Reply:
Can't you use the `relu` result as the mask?